pkg/cli-runtime/options.go

```diff
@@ -29,10 +29,12 @@ import (
 // validated. This function is typically used to define the PreRunE phase of a command.
 //
 // ```
-// cmd := &cobra.Command{
-// 	   ...
-// 	   PreRunE: cli.ValidateE(obj),
-// }
+//
+//	cmd := &cobra.Command{
```

Comment:
Remove changes in this file. It is not related to the PR

Reply:
Done